examples/website/i3s/normals-utils.js

```diff
@@ -20,9 +23,25 @@ export function generateBinaryNormalsDebugData(tile) {
   const normals = tile.content.attributes.normals.value;
   const positions = tile.content.attributes.positions.value;
   const modelMatrix = tile.content.modelMatrix;
+  const cartographicModelMatrix = tile.content.cartographicModelMatrix;
   const cartographicOrigin = tile.content.cartographicOrigin;
 
-  return {src: {normals, positions}, length: positions.length, modelMatrix, cartographicOrigin};
+  const cartesianPositions = new positions.constructor(positions.length);
+  for (let i = 0; i < cartesianPositions.length; i += 3) {
+    const position = positions.subarray(i, i+3);
+    scratchVector.set(position[0], position[1], position[2]);
+    scratchVector.transform(modelMatrix).add(cartographicOrigin);
+    Ellipsoid.WGS84.cartographicToCartesian(scratchVector, scratchVector);
```

Comment:
Again we need to reproject everything on the CPU?

Reply:
It is done to add normals correctly. I can do it only in cartesian. They are oriented in cartesian space.